client/src/partials/inventory/list/list.js

```diff
@@ -26,14 +27,23 @@ function InventoryListController ($translate, Inventory, Notify, uiGridConstants
 
   /** buttons in the headercrumb */
   vm.bcButtons = [
-    { icon: 'fa fa-filter', label: $translate.instant('FORM.BUTTONS.FILTER'), action: toggleFilter, color: 'btn-default' },
-    { icon: 'fa fa-plus', label: $translate.instant('FORM.LABELS.ADD'), action: addInventoryItem, color: 'btn-default' },
-    { icon: 'fa fa-print', label: $translate.instant('FORM.LABELS.PRINT'), action: printList, color: 'btn-default' }
+    { icon: 'fa fa-filter', label: $translate.instant('FORM.BUTTONS.FILTER'),
+      action: toggleFilter, color: 'btn-default'
+    },
+    { icon: 'fa fa-plus', label: $translate.instant('FORM.LABELS.ADD'),
+      action: addInventoryItem, color: 'btn-default',
+      dataMethod: 'create'
+    },
+    { icon: 'fa fa-print', label: $translate.instant('FORM.LABELS.PRINT'),
+      action: printList, color: 'btn-default'
+    }
   ];
 
   // edit button template
   var editTemplate = '<div style="padding: 5px;">' +
-    '<a title="{{ \'FORM.LABELS.EDIT\' | translate }}" href="" ng-click="grid.appScope.editInventoryItem(row.entity)">' +
+    '<a title="{{ \'FORM.LABELS.EDIT\' | translate }}" href="" ' +
+    'ng-click="grid.appScope.editInventoryItem(row.entity)" ' +
+    'data-edit-metadata="{{ row.entity.label }}">' +
```

Comment:
Instead of using the label, could you use the id?  That is guaranteed to be unique.